temperatureEndpoint.py

```diff
@@ -0,0 +1,62 @@
+from urllib.request import urlopen
+from urllib.error import URLError, HTTPError
+import json
+import abc
+
+class AbstractDataService(metaclass=abc.ABCMeta):
+    def get_weather_forecast(self):
+        pass
```

Comment:
specifically for abstract classes (interfaces) it's best practice to add documentation on the methods to explain what they do, their method parameters & return types